README.md

```diff
@@ -52,15 +52,11 @@ It is a lightweight microservices framework implements by golang. It provides ma
 
 ### **[erda-actions](https://github.com/erda-project/erda-actions)**
 
-It holds the definition for the official Pipeline Actions of the Erda Action Marketplace as found at https://www.erda.cloud/market/action. A Pipeline Action plays as a mininal runnable unit in [Erda Pipelines](https://docs.erda.cloud/1.0/manual/deploy/pipeline.html) such as checking out source code in github, buiding a docker image, creating a deployment in a Kubernetes cluster etc.. You can find more information about Pipeline Actions [here](https://docs.erda.cloud/1.0/manual/actions).
+It holds the definition for the official Pipeline Actions of the Erda Action Marketplace as found at https://www.erda.cloud/market/action. A Pipeline Action plays as a mininal runnable unit in [Erda Pipelines](https://docs.erda.cloud/1.2/manual/dop/concepts/pipeline.html) such as checking out source code in github, buiding a docker image, creating a deployment in a Kubernetes cluster etc.
```

Comment:
You should use latest.

`https://docs.erda.cloud/latest/manual/dop/concepts/pipeline.html`